main.cc

```diff
@@ -1021,6 +1021,16 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
             dbcfg.commitlog_scheduling_group = make_sched_group("commitlog", "clog", 1000);
             dbcfg.available_memory = memory::stats().total_memory();
 
+            // Make sure to initialize the scheduling group keys at a point where we are sure
+            // that nobody else will be creating service levels (e.g. service levels).
```

Comment:
repetition

Reply:
Fixed, hopefully the comment makes more sense now